.editorconifg

```diff
@@ -0,0 +1,14 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
```

Comment:
```suggestion
```
Remove local enforcement of line endings in favor of enforcing what goes into the repo via .gitattributes